packages/apps/src/Menu/index.tsx

```diff
@@ -162,9 +161,9 @@ function Menu ({ className = '' }: Props): React.ReactElement<Props> {
 export default React.memo(styled(Menu)(({ theme }: ThemeProps) => `
   align-items: center;
   display: flex;
-  justify-content: space-between;
   padding: 0;
   z-index: 220;
+  font-family: 'Nunito Sans',sans-serif;
```

Comment:
Please remove. We just use the global fond, it doesn't make sense to add these all-over the app wherever required.